### ISSUE TITLE
Is "Satchel of helpful goods" bugged?

### ISSUE THREAD

**Issue by Shieldborn** [dalaranwow/dalaran-wow#5330]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I did two dungeons as level 15 with Looking for group. When completed I didn't get any "Satchel of helpful goods". 
**Expected behaviour**: Tell us what should happen instead
I should have received one of them cause its one per day.
**Steps to reproduce the problem**:

1. Look if there is a bug with it.
2. 
3. 

**Include proofs for this behaviour**
I don't have any proof unless you can see in my gear or bags that I don't have any of those items usually received from those. But if you know how to fix the bugg that enough for me.
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
My character is a warrior "Shieldborn" on Algalon server. I first did Shadowfang Keep and then Deadmines.
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Zarjofgt** [comment]
It has to be a random dungeon, and its working as intended

**Comment by andreiacostinoaei** [comment]
If You disconnected and when you go back and get a reinv you won't get the satchel... Also the items from satchel are random and sometimes will not give you items :)))

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/29
